### PR TITLE
bump libxlsxwriter to 1.1.3

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **libxlsxwriter/1.1.3**

- move patch file for 1.0.0 to patches/1.0.0
- bump libxlsxwriter to 1.1.3
- add several options "tmpfile", "md5", "fmemopen", "dtoa"
    - md5=openssl is not supported in 1.0.0
---

- [+] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [+] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [+] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [+] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libxlsxwriter/all/conanfile.py [hunk 1]
```diff
@@ -12,11 +12,19 @@ class LibxlsxwriterConan(ConanFile):
...
+        "dtoa": [True, False],
     }
     default_options = {
         "shared": False, 
-        "fPIC": True
+        "fPIC": True,
+        "tmpfile": False,
+        "md5": "openwall",
+        "fmemopen": False,
+        "dtoa": False,
```
Comment (uilianries) [inline recipes/libxlsxwriter/all/conanfile.py:27]:
```suggestion
        "dtoa": False,
```

Trailing space

File: recipes/libxlsxwriter/all/conanfile.py [hunk 2]
```diff
@@ -30,16 +38,24 @@ def _source_subfolder(self):
...
 
+    def validate(self):
+        if tools.Version(self.version) <= "1.0.5" and self.options.md5 == "openssl":
+            raise ConanInvalidConfiguration("{0}:md5=openssl is not suppported in {0}/{1}".format(self.name, self.version))
+
     def requirements(self):
         self.requires("minizip/1.2.11")
         self.requires("zlib/1.2.11")
+        if self.options.md5 == "openssl":
+            self.requires("openssl/1.1.1c")
```
Comment (uilianries) [inline recipes/libxlsxwriter/all/conanfile.py:58]:
```suggestion
            self.requires("openssl/1.1.1l")
```

Can you the latest patch version?

**Review by uilianries — COMMENTED** [review]
I only concerned about the OpenSSL version, it would be great using the latest version if possible.

File: recipes/libxlsxwriter/all/conanfile.py (hunk 1 above)
Comment (toge) [reply recipes/libxlsxwriter/all/conanfile.py:27]: I'm sorry, it's my fault.

File: recipes/libxlsxwriter/all/conanfile.py (hunk 2 above)
Comment (toge) [reply recipes/libxlsxwriter/all/conanfile.py:58]: I agree with you. 
There are no reason to use old version.
Now I understand version policy in conan.

Comment (uilianries) [reply recipes/libxlsxwriter/all/conanfile.py]: Thank you! Some few projects really a specific version due API compatibility, fortunately, we don't have such case here.

**Review by uilianries — APPROVED** [review]

File: recipes/libxlsxwriter/all/conanfile.py (hunk 2 above)
Comment (toge) [reply recipes/libxlsxwriter/all/conanfile.py]: Thank you for detail information.
I understand.

**Review by prince-chrismc — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**